root/entity/Aliases.js

```diff
@@ -14,7 +14,7 @@ import AliasesComponent from '../components/Aliases/index.js';
 import chooseLayoutComponent from '../utility/chooseLayoutComponent.js';
 
 type Props = {
-  +aliases: $ReadOnlyArray<AliasT>,
+  +aliases: $ReadOnlyArray<AliasT<mixed>>,
```

Comment:
Would it make sense to have an `AliasTypeT` with the possible string values like we have for other `TypeT`s? I guess that'd clash with `AliasTypeT = OptionTreeT<'alias_type'>` - just feels weird to have `mixed` here.